common-test-lib/src/main/scala/com/gu/integration/test/util/CookieUtil.scala

```diff
@@ -11,6 +11,11 @@ object CookieUtil {
     driver.manage().deleteCookieNamed(cookieName)
   }
 
+  def createCookie(cookieName: String, content: String)(implicit driver: WebDriver) = {
+    val cookie = new Cookie(cookieName, content)
+    driver.manage().addCookie(cookie)
+  }
+
   /**
```

Comment:
this looks like something that could be useful to add to the framework, please create a PR here: https://github.com/guardian/scala-automation